src/LibraryViewExtension/web/library/layoutSpecs.json

```diff
@@ -766,7 +766,7 @@
               "elementType": "group",
               "include": [
                 {
-                  "path": "BuiltIn.LoopWhile"
+                  "path": "BuiltInNodes.LoopWhile"
```

Comment:
@mjkkirschner I had to make this change in order to continue displaying the `LoopWhile` node in the library. However this change made its icon disappear. Craig and I then had to create a new `.resx` file for it so that its icon could work again.

Reply:
@aparajit-pratap @mjkkirschner @saintentropy I was more so confused about both resx files living in the same project as opposed to putting the Dictionary icons resx `BuiltInImages.resx` in `BuiltIn.csproj`.  Something like this 
![image](https://user-images.githubusercontent.com/13341935/37175338-3a086ec4-22e7-11e8-8bc6-20d200841558.png)